### PR TITLE
sttaran // fixed the use of Locator options

### PR DESCRIPTION
**Pull request by StanislavTaran**
This implementation fix use LocatorOptions for Page and Locator.
Related issue is #274

### CODE REVIEW

File: locator.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ func newLocator(frame *frameImpl, selector string, options ...LocatorLocatorOpti
 			if frame != has.frame {
 				return nil, errors.New("inner 'has' locator must belong to the same frame")
 			}
-			selector += " >> has=" + has.selector
+			selector += fmt.Sprintf(` >> has="%s"`, has.selector)
```
Comment (mxschmitt) [inline locator.go:33]: Lets do it maybe the same was as upstream? https://github.com/microsoft/playwright/blob/634ba85c83ae19de1e1610e5b393e35eb03012e4/packages/playwright-core/src/client/locator.ts#L54

Comment (StanislavTaran) [reply locator.go:33]: sorry for the late reply, I have some urgent matters. Thanks for the comment, I will present a new version in the coming days

Comment (StanislavTaran) [reply locator.go]: @mxschmitt Yes, I think it was a good idea. I replaced fmt.Sprintf with  json.Marshal. Could you take a look one more time?

**Review by kumaraditya303 — APPROVED** [review]

**Review by mxschmitt — APPROVED** [review]

**Comment by mxschmitt** [comment]
Thanks!

**Pull request merged**